README.md

```diff
@@ -356,6 +357,7 @@ Table of Contents
    * https://www.visualstudio.com/en-us/products/visual-studio-community-vs.aspx - Visual Studio. Not only for Windows and .NET
    * https://codio.com/ - Codio is a cloud-based computer programming platform for universities, schools, and developer professionals.
    * http://www.stackhive.com/ - Cloud based IDE in browser that supports HTML5/CSS3/jQuery/Bootstrap
+   * http://www.sublimetext.com/3 - Sublime Text is a sophisticated text editor for code, markup and prose.
```

Comment:
i believe the other here are hosted or very tightly coupled with a hosted component, sublime is mainly a download standalone app am I right?

If that's the case I'd rather not list it